pkg/apis/flow/v1alpha1/xmltojsontransformation_lifecycle.go

```diff
@@ -29,7 +29,18 @@ const (
 	EventTypeXMLToJSONGenericResponse = "io.triggermesh.xmltojsontransformation.error"
 )
 
-var xmlToJSONCondSet = apis.NewLivingConditionSet()
+const (
+	// ConditionReady has status True when the router is ready to send events.
+	ConditionReady = apis.ConditionReady
+	// ConditionSinkProvided has status True when the router has been configured with a sink target.
+	ConditionSinkProvided apis.ConditionType = "SinkProvided"
+)
+
+var XMLToJSONCondSet = apis.NewLivingConditionSet(
```

Comment:
Does this type need to be exported?

Reply:
you know... i wondered the same thing. I copied a lot of this from `Synchronizer` he had it exported. so I left it exported.

Reply:
Ok but, does it need to be? Is anything outside of this package using it?

Reply:
nope! I have fixed it .